python/fledge/services/common/microservice.py

```diff
@@ -118,6 +118,10 @@ def _get_service_registration_payload(self):
     async def shutdown(self, request):
         pass
 
+    @abstractmethod
+    async def restart(self, request):
+        pass
+
```

Comment:
Also I believe we do not need this, as restart is not common to all micro services. Instead only with core.